### ISSUE TITLE
Device Support for Bintec RS120

### ISSUE THREAD

**Issue by jgadm** [librenms/librenms#6765]
DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

GitHub issues is for  bugs, please do not post issues asking for help or how to do X, Y or Z. 
You can use our irc channel ##librenms on freenode to ask questions or our [community site](https://community.librenms.org).
If you have a feature request please post this on our [community site](https://community.librenms.org/c/feature-requests).

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x ] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [x ] Please include all of the information between the `====================================` section of `./validate.php` which you can run from the cli.
```
Component | Version
--------- | -------
LibreNMS  | f8aadf227b25ff4436859d19062d234ed9134767
DB Schema | 192
PHP       | 5.6.30-0+deb8u1
MySQL     | 5.5.54-0+deb8u1
RRDTool   | 1.4.8
SNMP      | NET-SNMP 5.7.2.1
```

- [x ] Please provide ALL info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [x ] Please provide as much detail as possible.
- [ x] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

Hi,

We have several Bintec R120's deployed and are not able to successfully polled by LibreNMS.  I see that support for Bintec SNMP MIB has been added with issue 2178, but the device is detected as generic and all of our interfaces are not detected as it errors with "Error: OID not increasing".

If we manually run snmpwalk with the -Cc option, we get a list of all interfaces. 

Unfortunately, we are not able to query the device with snmpbulkwalk as it returns "Request timed out" and the device logs "error in packet from IP:PORT".

Looking at includes/definitions/bintec-smart.yaml, it has sysObjectID .1.3.6.1.4.1.272.4.201.82.78.79.48, but our devices return .1.3.6.1.4.1.272.4.201.83.82.49.48.0.0

I have attached the outputs of:

 ./discovery.php -h gateway
 ./poller.php -h gateway -r -f -d
 snmpwalk -v2c -c $COMMUNITY_STRING gateway
 snmpwalk -v2c -Cc -c $COMMUNITY_STRING gateway

Can you kindly advise on how to troubleshoot this further to add device compatibility?

Many thanks
James

[snmpwalk.txt](https://github.com/librenms/librenms/files/1044386/snmpwalk.txt)
[snmpwalk-cc.txt](https://github.com/librenms/librenms/files/1044387/snmpwalk-cc.txt)
[discovery.txt](https://github.com/librenms/librenms/files/1044392/discovery.txt)
[poller.txt](https://github.com/librenms/librenms/files/1044391/poller.txt)

**Comment by laf** [comment]
I've submitted a fix to have these detected. As for the timeouts, you should try changing the snmp timeouts in edit device -> snmp. However if you have further issues with that then please stop by the irc channel or our community site for further help

**Comment by jgadm** [comment]
Hi laf,

Thanks!
With regards to the interfaces not being detected, I tried the proposed solution, but the issue appears to be with the OIDs being returned out of order. Is it possible to add the -Cc option with this device for discovery? I've attached the output of snmpwalk with and without the -Cc option.

[snmpwalk.txt](https://github.com/librenms/librenms/files/1054900/snmpwalk.txt)


Best regards,
James
